conf/machine/include/imx8x-mek.inc

```diff
@@ -0,0 +1,70 @@
+MACHINEOVERRIDES =. "mx8:mx8x:"
+
+require conf/machine/include/imx-base.inc
+require conf/machine/include/tune-cortexa35.inc
+
+IMX_DEFAULT_BSP = "nxp"
+
+MACHINE_FEATURES += "pci optee bcm43455 bcm4356"
+MACHINE_FEATURES_append_use-nxp-bsp = " bcm4359"
+
+# Don't include kernels in standard images
+RDEPENDS_${KERNEL_PACKAGE_NAME}-base = ""
+
+LOADADDR = ""
+
+# We have to disable SERIAL_CONSOLE due to auto-serial-console
+SERIAL_CONSOLES = "115200;ttyAMA0"
+
+# we do not want to have getty running on tty1 as we run
+# auto-serial-console there
+USE_VT = "0"
+
+KERNEL_DEVICETREE = " \
+    freescale/${KERNEL_DEVICETREE_BASENAME}.dtb \
```

Comment:
This header does not have a default definition for `KERNEL_DEVICETREE_BASENAME`, unlike _conf/machine/include/imx8mm-evk.inc_ and _conf/machine/include/imx8mn-evk.inc_.

Can this be made consistent? Please see my comments in headers mentioned.